APIKit.podspec

```diff
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
   }
 
   s.ios.deployment_target = "8.0"
-  s.osx.deployment_target = "10.10"
+  s.osx.deployment_target = "10.11"
```

Comment:
Looks like you were hitting https://openradar.appspot.com/49262697 but this has been addressed in #270 (temporarily). So Please revert these deployment target changes.